resources/user-guides/launch-data-elements.md

```diff
@@ -0,0 +1,86 @@
+# Data Elements
+
+Data elements are the building blocks for your data dictionary and are used to collect, organize, and deliver data across marketing and ad technology.
+
+A data element is a variable where the value can be mapped to data within the Experience Platform Mobile SDK such as visitor ID, device name, user profile, number of launches, et cetera. In Experience Platform Launch you can reference this value by its variable name. This collection of data elements becomes the dictionary of defined data that you can use to build rules for your application, and this dictionary is shared across Experience Platform Launch where it can be used with any extension added to your property.
```

Comment:
- In the 1st sentence:
Please replace "within" with "in".
Please add "a" before "visitor iD", "device name", and "user profile".
Please replace "et cetera" with "and so on".
- In the 2nd sentence, please add a comma after "Experience Platform Launch"
- In the 3rd sentence, please add "that is" between "extension" and "added".